src/firefly/js/charts/ui/PlotlySaveDialog.jsx

```diff
@@ -63,25 +64,24 @@ async function saveFile(request, Plotly, chartDiv) {
 
 const PlotLySavePanel= function( {isWs,Plotly, chartDiv, filename}) {
     return (
-        <Stack p={1} flexGrow={1} justifyContent='space-between'>
-            <FieldGroup groupKey={'PlotLySaveField'} >
+        <FieldGroup groupKey={'PlotLySaveField'} sx={{height: 1}}>
+            <Stack p={1} justifyContent='space-between' spacing={2} height={1}>
                 <ValidationField
                     fieldKey={'filename'}
                     initialState= {{
                         value: filename,
                         tooltip: 'Enter filename of chart png',
                         label: 'Chart Filename',
                     }} />
-            </FieldGroup>
-
-            <Stack flexDirection='row' justifyContent='space-between'>
-                <Stack spacing={1} direction='row' alignItems='center'>
-                    <CompleteButton text='Save' dialogId={DIALOG_ID}
-                                    onSuccess={(request) => saveFile(request,Plotly, chartDiv)} />
-                    <Button onClick={() => dispatchHideDialog(DIALOG_ID)}>Cancel</Button>
+                <Stack flexDirection='row' justifyContent='space-between'>
```

Comment:
Can you change `flexDirection` to `direction`.  I know they are everywhere, but let's fix them when we encounter them.

Reply:
sure - overlooked it!